src/components/AssetImage/AssetImage.css

```diff
@@ -1,12 +1,12 @@
-.AssetImage {
+.uiAssetImage {
```

Comment:
Maybe we can add `Dui` for decentraland styles, like material does but with `Mui`